OutaTime/OutaTime/DatePickerViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  DatePickerViewController.swift
+//  OutaTime
+//
+//  Created by Lambda_School_Losaner_256 on 1/29/20.
+//  Copyright © 2020 Benglobal Creative. All rights reserved.
+//
+
+import UIKit
+protocol  DatePickerDelegate {
+    func destinationDateWasChosen(date: Date)
+}
+class DatePickerViewController: UIViewController {
+    @IBOutlet weak var datePicker: UIDatePicker!
+    @IBAction func doneTapped(_ sender: UIBarButtonItem) {
```

Comment:
Needed to add this step in the done action here...
"call the delegate method to inform the delegate of the new date chosen by the user"
And then you call dismiss. It looks like this:
`delegate?.destinationDateWasChosen(datePicker.date)
        dismiss(animated: true, completion: nil)`